packages/aws-cdk-lib/aws-apigatewayv2-integrations/lib/websocket/aws.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  WebSocketRouteIntegration,
+  WebSocketIntegrationType,
+  WebSocketRouteIntegrationConfig,
+  WebSocketRouteIntegrationBindOptions,
+} from '../../../aws-apigatewayv2';
+import { IRole } from '../../../aws-iam';
+
+/**
+ * Props for AWS type integration for an HTTP Api.
+ */
+export interface WebSocketAwsIntegrationProps {
+  /**
+   * Integration URI.
+   *
+   * @default None.
```

Comment:
Nit: I don't like any of these `@default None.` It doesn't tell me any information for what it means to not provide the property. Moreover, for some of these I don't believe the default truly is "none." We should be describing the default behavior better for all these props.